player/build.gradle

```diff
@@ -2,16 +2,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-  compileSdkVersion versions.androidSdk.compile
-
-  defaultConfig {
-    minSdkVersion versions.androidSdk.min
-    targetSdkVersion versions.androidSdk.target
-  }
+    compileSdkVersion versions.androidSdk.compile
 }
 
 dependencies {
-  implementation project(":no-player")
-  implementation libraries.androidxAppCompat
-  implementation libraries.kotlinStdLib
+    implementation project(':no-player')
+
+    implementation libraries.androidxAppCompat
+    implementation libraries.kotlinStdLib
```

Comment:
we should probably add project formatting to the repo else there'll be a lot of back and forth 😬

Reply:
💯 completely agree, I'm easy stylewise although I tend for lean towards line per parameter when needed

```
val bar = foo(a, b, c)
val realllyLongVar = myReallyLongFunction(
    longParameter = "foobar",
    longParameter2 = "foobar"
)

```

Reply:
alt-enter in AS lets you toggle between 1 line and multiple lines (as you've written)